content/docs/user-guide/contributing/blog.md

```diff
@@ -29,9 +29,9 @@ descriptionLong: |
   Some long
   multiline
   text (supports _basic_ [Markdown](link))
-picture: /uploads/image.jpeg
```

Comment:
let's put something similar to what we have in the current posts - the are organized by date?

Reply:
Great point!